src/System.Windows.Forms/src/System/Windows/Forms/Dialogs/TaskDialog/TaskDialogButton.cs

```diff
@@ -413,13 +411,9 @@ internal bool HandleButtonClicked()
         return text;
     }
 
-    internal ComCtl32.TDCBF GetStandardButtonFlag()
-    {
-        if (!IsStandardButton)
-            throw new InvalidOperationException();
-
-        return GetStandardButtonFlagForResult(_standardButtonResult!.Value);
-    }
+    internal TASKDIALOG_COMMON_BUTTON_FLAGS GetStandardButtonFlag() => !IsStandardButton ?
+        throw new InvalidOperationException() :
+        GetStandardButtonFlagForResult(_standardButtonResult!.Value);
```

Comment:
nit: `? :` should be at start of line
```suggestion
    internal TASKDIALOG_COMMON_BUTTON_FLAGS GetStandardButtonFlag() => !IsStandardButton 
        ? throw new InvalidOperationException()
        : GetStandardButtonFlagForResult(_standardButtonResult!.Value);
```